contracts/UFragmentsPolicy.sol

```diff
@@ -79,6 +74,8 @@ contract UFragmentsPolicy is Ownable {
 
     uint256 private constant DECIMALS = 18;
 
+    int256 private constant ONE = int256(10**DECIMALS);
```

Comment:
Just checking that this does not change the contract memory layout.

(I don't think it does, but `immutable` would?)

Reply:
constants dont go into storage as far as I remember